### DISCUSSION TITLE
MinGW BuildBot

### DISCUSSION THREAD

**Discussion by ghost** [halide/Halide#5348] [Q&A] [answered]
To whom it may concerns,
I wonder if the MinGW 64 buildbot is still included in the master or any other specific branch and hopefully if yes, then really appreciate redirecting me there in order to build it successfully.
Best Regards,
Ramin

**Comment by steven-johnson** [comment, marked as answer]
Sorry, we no longer actively support MinGW as a build environment. We would consider patches that provide compatibility but can make no guarantee that it will stay unbroken.

**Reply by ramin-raeisi** [reply]
No worries as I can plan to change to the MSVC but it is not easy, so I wonder if you can redirect me to a specific branch or commit in which you have the MinGW buildbot so I can use it for evaluation and quick start purposes?

**Comment by steven-johnson** [comment]
The support was removed quite a while ago; the code has changed substantially enough that it would be of little help.

You might have an easier time just using the WSL subsystem on Windows.
